docs/data/date-pickers/custom-components/PaperContentComponent.js

```diff
@@ -0,0 +1,203 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
```

Comment:
We migrated to `dayjs` on our doc examples (see #5481), the diff should be pretty straightforward, you have examples in the adapter in the `addWeeks` etc... methods if you want

Reply:
Thanks for reminding.
Did sync with master and refactor to use `dayjs` 😉